docs/releases/v1.2.0.md

```diff
@@ -0,0 +1,114 @@
+# Releases notes
+
+## Changelog
+
+Changes between `1.1.0` and this release: `1.2.0`.
+
+- Removed all `ark` deployments. See upgrade procedure to understand how to update from **Ark** to **Velero**.
+- Update [Velero](../../katalog/velero) version from *v1.0.0* to *v1.2.0* version.
+- Add [on premises velero deployment](../../katalog/velero/velero-prem).
+- Add [E2E testing](../../.drone.yml).
+
+
+## Upgrade procedure
+
+### From Ark version 0.10
+
+With a `kustomization.yaml` file like:
+
+```yaml
+namespace: kube-system
+
+bases:
+  - katalog/ark-{aws,azure,gcp}
+  - katalog/akr-restic-{aws,azure,gcp} # Optional
+  - katalog/ark-schedule # Optional
+```
+
+Rename it to `kustomization.yaml.old` and create a new `kustomization.yaml` with the following bases:
```

Comment:
Should we write down that `kustomization.yaml.old` needs to be deleted once the migration is done?

Reply:
I've added a `rm` command at the end of the procedure:

```bash
.
.
.
# Remove old Ark data
$ kubectl delete crds -l component=ark
$ kubectl delete clusterrolebindings -l component=ark
$ kubectl delete deploy ark -n kube-system
$ kubectl delete ServiceMonitor ark -n kube-system

# Remove old kustomization.yaml.old file 
$ rm kustomization.yaml.old
```

Thanks for the question/suggestion ;)